example/views/index.jsx

```diff
@@ -8,8 +9,19 @@ import {SubNavBar, NavItem} from '../../components/subnavbar';
 import {Button, LinkButton, InputButton} from "../../components/button";
 import ExpandableText from '../../components/typography/expandable-text';
 import OptionSelect, {CheckedOption} from '../../components/option-select';
-import { TabBar } from '../../components/tabbar'
-import { TabItem } from '../../components/tabbar/tabitem'
+import { TabBar } from '../../components/tabbar';
+import { TabItem } from '../../components/tabbar/tabitem';
+import DataTable from '../../components/datatable';
+import places from '../data/places';
+import schema from '../schema/places';
+
+const formatters = {
+  suitability: { format: arr => arr.join(', ') },
+  holding: { format: arr => arr.join(', ') },
+  nacwo: { title: key => key.toUpperCase() }
```

Comment:
Again, can we come up with a _super_ generic example dataset?

Reply:
yeppo